test/integration_tests/test_similarity.py

```diff
@@ -0,0 +1,152 @@
+# coding=utf-8
+# Copyright 2018-2022 EVA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+import unittest
+from test.util import create_sample_image, load_inbuilt_udfs
+
+import numpy as np
+import pandas as pd
+
+from eva.catalog.catalog_manager import CatalogManager
+from eva.configuration.configuration_manager import ConfigurationManager
+from eva.models.storage.batch import Batch
+from eva.server.command_handler import execute_query_fetch_all
+from eva.storage.storage_engine import StorageEngine
+
+
+class SimilarityTests(unittest.TestCase):
```

Comment:
We should consider adding a test case using the `FeatureExtractor` UDF.

Reply:
Fix this.